src/infrastructure/figma/figma-service.ts

```diff
@@ -204,17 +204,22 @@ export class FigmaService {
 			);
 
 			if (!devResourceToDelete) {
-				getLogger().error(
+				getLogger().info(
```

Comment:
**note:** Log as "info" since it is a valid business use case (a user can manually delete a _Dev Resource_ from a Figma _File_ anytime).